tests/common/runner.rs

```diff
@@ -20,11 +19,24 @@ pub fn mock_account(idx: usize) -> (Address, PlainAccount) {
     )
 }
 
-// Build an in memory storage for execution.
+// Build InMemoryStorage.
 pub fn build_in_mem(
     accounts: impl IntoIterator<Item = (Address, PlainAccount)>,
 ) -> InMemoryStorage {
-    accounts.into_iter().collect::<ChainState>().into()
+    build_in_mem_with_block_hashes(accounts, [])
+}
+
+// Build InMemoryStorage with block hashes.
+pub fn build_in_mem_with_block_hashes(
```

Comment:
Our use cases are too small to justify this complexity (new dev asking which function to call for a new test). I'd add `, []` to the all existing call sites.

I think we can improve the existing `build_in_mem` name, too. `build_in_mem_storage` or just `build_storage`?

Reply:
I will change the function signature of `InMemoryStorage::new` to:

```
pub fn new(
        accounts: impl IntoIterator<Item = (Address, impl Into<InMemoryAccount>)>,
        block_hashes: impl IntoIterator<Item = (U256, B256)>,
    ) -> Self
```